CIP-0048/README.md

```diff
@@ -0,0 +1,249 @@
+---
+CIP: 48?
+Title: 721 References and Payloads
+Authors: Jack (ada9000, https://forum.cardano.org/u/jack7e/, @ada9000_ twitter)
+Comments-URI: https://forum.cardano.org/t/cip-extended-721/
+Status: Draft
+Type: Informational
+Created: 2022-04-22 (updated 2022-09-22)
+---
+
+## Important note
+
+This CIP mentions the `ext` json key defined by [CIP49](https://github.com/cardano-foundation/CIPs/pull/343/files) which is an outcome of work on CIP48.
+
+# Abstract
```

Comment:
```suggestion
## Abstract
```
H2 section titles make room for an H1 title which will be added programmatically from the title in the preamble. https://github.com/cardano-foundation/CIPs/blob/master/CIP-0001/README.md#structure